.github/workflows/cypress.yml

```diff
@@ -0,0 +1,47 @@
+name: Cypress
+on: [push]
+jobs:
+  cypress:
+    name: Cypress
+    runs-on: ubuntu-18.04
+    env:
+      MAAS_PORT: 5240
+    steps:
+      - uses: actions/checkout@main
+      - name: Set environment variables
+        run: echo "::set-env name=IP_ADDRESS::$(hostname -I | cut -d" " -f1)"
+      - name: Install MAAS
+        run: |
+          sudo systemctl enable snapd
+          sudo snap install maas --channel=latest/edge
+          sudo snap install maas-test-db
+      - name: Initialise MAAS
+        run: |
+          sudo maas init region+rack \
+            --maas-url=http://${{env.IP_ADDRESS}}:${{env.MAAS_PORT}}/MAAS \
+            --database-uri maas-test-db:///
+      - name: Create MAAS admin
+        run: sudo maas createadmin --username=admin --password=test --email=fake@example.org
+      - name: Start MAAS
+        run: sudo snap start maas
```

Comment:
Why do you (re)start maas here?

MAAS is already running from the `Install MAAS` step

Reply:
An initial attempt to diagnose why maas wasn't running. Don't intend to preserve this. 

This appears to be the problem:

```
2020-08-26 04:28:03 maasserver.start_up: [error] Database error during start-up
Traceback (most recent call last):
  File "/snap/maas/8804/usr/lib/python3/dist-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "metadataserver_script" does not exist
LINE 1: ...server_script"."apply_configured_networking" FROM "metadatas...
```

which seems odd to me as the migration have definitely run. See https://github.com/canonical-web-and-design/maas-ui/runs/1029790811?check_suite_focus=true for details

Reply:
Turns out this was a red herring, although perhaps it might be worth looking into the exception?